skimage/feature/util.py

```diff
@@ -67,6 +68,10 @@ def plot_matches(ax, image1, image2, keypoints1, keypoints2, matches,
         color is chosen randomly.
     only_matches : bool, optional
         Whether to only plot matches and not plot the keypoint locations.
+    alignment : str, optional
```

Comment:
I think the common format for such kind of parameters in scikits is the following:
```
alignment : {'horizontal', 'vertical'}, optional
```